### PR TITLE
Fix/action menu styles

### LINKED ISSUE #953: action menu styles not building correctly
CSS file built by action menu is missing `.psds-actionmenu` class. Last reported working on 9.1.2. by Carolina Powers

### PR DESCRIPTION
**Pull request by EdwardIrby**
resolve #953

### CODE REVIEW

File: packages/actionmenu/src/react/menu.js [hunk 1]
```diff
@@ -16,7 +17,11 @@ const slide = css.keyframes(
 const styles = {
   menu: ({ _isNested, origin }) => {
     return compose(
-      css(stylesheet['.psds-actionmenu']({ slide })),
+      css(stylesheet['.psds-actionmenu']),
+      css({
+        animation: `${slide} ${motion.speedNormal} forwards`
+      }),
+      css(stylesheet['@keyframes psds-actionmenu__keyframes__slide']),
```
Comment (jaketrent) [inline packages/actionmenu/src/react/menu.js:24]: What does including this line here accomplish?

**Comment by danethurber** [comment]
is the build fix simple to apply. i'd rather fix it there

File: packages/actionmenu/src/react/menu.js (hunk 1 above)
Comment (EdwardIrby) [reply packages/actionmenu/src/react/menu.js:24]: `'.psds-actionmenu': ({ slide }) => ({   ` does not build css with the class `'.psds-actionmenu`. That enables both the react code to function and the css to build correctly. I can try and fix the build process but it doesn't properly handle the examples you posted above.

**Comment by EdwardIrby** [comment]
> is the build fix simple to apply. i'd rather fix it there
@danethurber I'll try and dix the build process instead of refactoring the files.

**Comment by jaketrent** [comment]
Here's a proposal on the fix via build: https://github.com/pluralsight/design-system/pull/964

File: packages/actionmenu/src/react/menu.js (hunk 1 above)
Comment (jaketrent) [reply packages/actionmenu/src/react/menu.js:24]: Huh, that's unexpected.  I would have thought that the usage of `glamor.css.keyframes` on line 13 and the application of that on line 22 in `animation` would have been enough to make the React component work.  And then the CSS build doesn't use this file at all, just `css/index.js`.

**Pull request closed without merging**